test/opensearch-cluster-cdk.test.ts

```diff
@@ -1070,3 +1070,50 @@ test('Ensure target group protocol is always TCP', () => {
     TargetType: 'instance',
   });
 });
+
+
+describe.each([
+  { loadBalancerType: 'alb', securityDisabled: false, expectedType: 'application', expectedProtocol: 'HTTPS' },
+  { loadBalancerType: 'alb', securityDisabled: true, expectedType: 'application', expectedProtocol: 'HTTP' },
+  { loadBalancerType: 'nlb', securityDisabled: false, expectedType: 'network', expectedProtocol: 'TLS' },
+  { loadBalancerType: 'nlb', securityDisabled: true, expectedType: 'network', expectedProtocol: 'TCP' },
+])('Test $loadBalancerType creation with securityDisabled=$securityDisabled', ({ loadBalancerType, securityDisabled, expectedType, expectedProtocol }) => {
```

Comment:
Can you add one case for throwing exception when unexpected input is provided?

Reply:
Added